mlflow/protos/service.proto

```diff
@@ -55,6 +55,17 @@ service MlflowService {
     };
   }
 
+  rpc getExperimentByName (GetExperimentByName) returns (GetExperimentByName.Response) {
+    option (rpc) = {
+      endpoints: [{
+        method: "GET",
+        path: "/preview/mlflow/experiments/get-name"
```

Comment:
Nit: this endpoint name sounds like we're getting the name of the experiment, maybe we could do `/preview/mlflow/experiments/get-by-name`?

Reply:
fixed :)